pkgs/by-name/sw/sway-unwrapped/package.nix

```diff
@@ -11,19 +11,16 @@
 , trayEnabled ? systemdSupport
 }:
 
-let
-  wlroots = wlroots_0_16;
```

Comment:
might make sense to already pin it to the current wlroots version as sway will probably have to release a new version to follow upstream wlroots changes?

Reply:
Should pin to `wlroots_0_17`, there will always be breaking changes before wlroots 1.0 is released

https://github.com/NixOS/nixpkgs/pull/245704#pullrequestreview-1553286765

Reply:
probably makes sense, indeed the convention seems to be to update wlroots with a next version of sway